packages/eth-json-rpc-provider/src/safe-event-emitter-provider.ts

```diff
@@ -26,11 +26,9 @@ type Eip1193Request<Params extends JsonRpcParams> = {
  * @param eip1193Request - The EIP-1193 request to convert.
  * @returns The corresponding JSON-RPC request.
  */
-export function convertEip1193RequestToJsonRpcRequest<
-  Params extends JsonRpcParams,
->(
-  eip1193Request: Eip1193Request<Params>,
-): JsonRpcRequest<Params | Record<never, never>> {
+export function convertEip1193RequestToJsonRpcRequest(
+  eip1193Request: Eip1193Request,
+): JsonRpcRequest {
```

Comment:
Same as above, this function needs to be generic over `Params`. Exposing no way to narrow the `params` property of the return object beyond `JsonRpcParams` would make this function unusable downstream.